src/frisby/spec.js

```diff
@@ -198,7 +198,7 @@ class FrisbySpec {
           this._doneFn = null;
         }
       } else {
-        result = fn(responseBody);
+        result = fn(responseBody, this._response.headers);
```

Comment:
Promises traditionally only take a single parameter, so I don't want to break that expectation by adding a seconf one here.